clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -311,6 +311,7 @@ void SYCL::fpga::BackendCompiler::ConstructJob(
 
   InputInfoList ForeachInputs;
   InputInfoList FPGADepFiles;
+  StringRef createdReportName;
```

Comment:
(Optional) suggestion for renaming as per LLVM standards along the way:
```suggestion
  StringRef CreatedReportName;
```

Reply:
Done